### PR TITLE
Avoid `import numpy`

### PR DESCRIPTION
**Pull request by okuta**
`import numpy` is executed every time in `a_cpu.ctypes.get_as_parameter()`

### CODE REVIEW

**Comment by kmaehashi** [comment]
Jenkins, test this please.

File: cupy/core/core.pyx [hunk 1]
```diff
@@ -1361,7 +1361,7 @@ cdef class ndarray:
             else:
                 a_gpu = self
             a_cpu = numpy.empty(self._shape, dtype=self.dtype, order=order)
-        ptr = a_cpu.ctypes.get_as_parameter()
```
Comment (kmaehashi) [inline cupy/core/core.pyx]: Is it better to remove other use of `get_as_parameter()` in `core.pyx`?

**Comment by okuta** [comment]
I fixed.
jenkins, test this please.

**Comment by kmaehashi** [comment]
LGTM!

**Pull request merged**